test/conftest.py

```diff
@@ -19,4 +21,19 @@ def default_text_runner(default_rules_collection):
     return RunFromText(default_rules_collection)
 
 
-# vim: et:sw=4:syntax=python:ts=4:
+def run_ansible_lint(cwd, role_path=None, bin=None, env=None):
```

Comment:
Conftest is not the best location for such helpers. It should have pytest-related things like fixtures and hooks. Put this into `__init__.py` along with other helpers instead of having them spread across multiple locations w/o any structure.

Also, this change is unrelated to GHA annotations and must be submitted separately.